### DISCUSSION TITLE
2023/05/12/%E9%94%90%E8%AF%84/%E8%A3%85%E7%94%B2%E6%82%AA%E9%AC%BC%E6%9D%91%E6%AD%A3/

### DISCUSSION THREAD

**Discussion by dangokanako** [dangokanako/blog-comments-gitalk#18] [Announcements]
https://dangokanako.github.io/2023/05/12/%E9%94%90%E8%AF%84/%E8%A3%85%E7%94%B2%E6%82%AA%E9%AC%BC%E6%9D%91%E6%AD%A3/ 

 document.addEventListener('DOMContentLoaded', function () {         const ap = new APlayer({             theme: '#FADFA3',             container: document.getElementById('my-aplayer1'),

**Comment by Algebraic-Zephyr** [comment]
Markdown Test: 
- TEST1
-  TEST2
Have a nice day, Dango chan
